server/graphql/types/src/types/item.rs

```diff
@@ -439,7 +438,6 @@ mod test {
                volumePerOuterPack
                margin
                weight
-               strength
```

Comment:
fix from previous PR, strength is now being translated rather than being parsed directly from the legacy record